background/load-addon-manifests.js

```diff
@@ -239,4 +239,18 @@ const localizeSettings = (addonId, setting, tableId) => {
   }
   scratchAddons.localState.ready.manifests = true;
   scratchAddons.localEvents.dispatchEvent(new CustomEvent("manifestsReady"));
-})();
+}
+loadAddonManifests();
+
+// Data reload required when toggling the forceEnglish setting
+chrome.runtime.onMessage.addListener(async function (request, sender, sendResponse) {
+  if (request === "reloadAddonManifests") {
+    // Only attempt to reload manifests if they are already loaded
+    if (!scratchAddons.localState.ready.manifests) return;
+
+    scratchAddons.localState.ready.manifests = false;
+    scratchAddons.manifests = [];
+    await loadAddonManifests();
+    sendResponse("done");
```

Comment:
It looks like this respose is never received.
```
Unchecked runtime.lastError: The message port closed before a response was received.
```

Reply:
It is not necessary to send a response.

If you really want to send one, then you need to return true; and avoid the use of async/await in favor of normal promises. (this is how asynchronously sending a response works with the chrome messaging API)